source/shared/cpp/ObjectModel/BaseElement.h

```diff
@@ -84,6 +84,12 @@ namespace AdaptiveSharedNamespace
         // Fallback and Requires support
         FallbackType GetFallbackType() const { return m_fallbackType; }
         std::shared_ptr<BaseElement> GetFallbackContent() const { return m_fallbackContent; }
+        void SetFallbackType(FallbackType type) { m_fallbackType = type; }
+        void SetFallbackContent(std::shared_ptr<BaseElement> element)
+        {
+            m_fallbackContent.reset();
```

Comment:
>m_fallbackContent.reset(); [](start = 12, length = 26)

Is this needed?

Reply:
suppose not :)

---
In reply to: [263172385](https://github.com/Microsoft/AdaptiveCards/pull/2504#discussion_r263172385) [](ancestors = 263172385)